hedera-node/hedera-network-admin-service-impl/src/main/java/module-info.java

```diff
@@ -22,15 +23,13 @@
     requires org.apache.commons.io;
     requires org.apache.logging.log4j;
     requires static com.github.spotbugs.annotations;
-    requires static java.compiler; // javax.annotation.processing.Generated
 
     provides com.hedera.node.app.service.networkadmin.FreezeService with
             FreezeServiceImpl;
     provides NetworkService with
             com.hedera.node.app.service.networkadmin.impl.NetworkServiceImpl;
 
-    exports com.hedera.node.app.service.networkadmin.impl to
-            com.hedera.node.app;
+    exports com.hedera.node.app.service.networkadmin.impl;
```

Comment:
We don't typically export `impl` packages to any other module besides `com.hedera.node.app`. Why is this necessary?

Reply:
Now I'm really confused...`WritableFreezeStore` and `FreezeUpgradeActions` are in the same (network.admin.impl) module...

I'm dying to know why we don't have to do this in other modules, e.g. consensus.impl. The `module-info.java` file there reads:
```
    exports com.hedera.node.app.service.consensus.impl to
            com.hedera.node.app;
    exports com.hedera.node.app.service.consensus.impl.handlers;
    exports com.hedera.node.app.service.consensus.impl.records;
    exports com.hedera.node.app.service.consensus.impl.schemas;
```

Then again, there are a handful of `module-info.java` files that do export their `.impl` packages without restriction. 

I guess it's fine since it's done in other places??

Reply:
It was strange to me as well. I also tried with `provides` clauses to give visibility but still no success